runtime/taskcontext.go

```diff
@@ -63,12 +73,14 @@ type TaskInfo struct {
 // properties, and abortion notifications.
 type TaskContext struct {
 	TaskInfo
-	webHookSet *webhookserver.WebHookSet
-	logStream  *stream.Stream
-	mu         sync.RWMutex
-	queue      client.Queue
-	status     TaskStatus
-	cancelled  bool
+	webHookSet  *webhookserver.WebHookSet
+	logStream   *stream.Stream
+	logLocation string
```

Comment:
Is `logLocation` an absolute path, a relative path to task home directory, does it have engine-specific path separator or a canonical `/` separator, etc? Can you add comments for these struct members?